### PR TITLE
Fix "attach links" for pjax loads

### LINKED ISSUE #6: no attach link added if you don't load the PR page
Steps to reproduce:

1. go to the page listing all pull requests. for example: https://github.com/mozilla-services/socorro/pulls
2. click on a pull request

Expected: An attach link shows up.

Actual outcome: No attach link shows up.

@Osmose pointed out that when you click on one of the pull request links, it's not actually doing a page load, but rather changing the existing page. So rob-bugson's stuff doesn't kick in. Maybe content-scripts don't trigger in that case?

### PR DESCRIPTION
**Pull request by willkg**
Fixes #6

### CODE REVIEW

**Comment by willkg** [comment]
@Osmose This fixes rob-bugson to work with pjax loads. I looked at octotree and they're doing essentially the same thing. Does this look ok and work for you?

File: github-pr.js [hunk 1]
```diff
@@ -77,31 +78,63 @@ function getAttachLinks(bugIds, prURL, prNum, prTitle) {
...
 
 /**
- * Creates the "Attach to bug: " text and link nodes and returns them.
+ * Checks if there's already a container and if not, creates one with attach
+ * links in it.
  */
-function createAttachLinks() {
+function createAttachLinksContainer() {
+    // If this is not a pull request page, then return.
+    if (! /^https:\/\/github.com\/[^\/]+\/[^\/]+\/pull\//.test(window.location.href)) {
```
Comment (Osmose) [inline github-pr.js]: For future reference, the [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) object makes parsing and testing URLs way easier than regex magic.

**Review by Osmose — APPROVED** [review]
Looks great, works for me when testing locally. Thanks for the fix! :D

**Comment by willkg** [comment]
@Osmose ^^^ Is that what you were thinking of when you said "the URL object ..."?

**Comment by Osmose** [comment]
> @Osmose ^^^ Is that what you were thinking of when you said "the URL object ..."?

Yup! So much easier to read. 👍

**Comment by willkg** [comment]
Thank you!

**Pull request merged**